lib/LedgerSMB/PSGI.pm

```diff
@@ -227,6 +227,8 @@ sub setup_url_space {
              pod_view => 'Pod::POM::View::HTMl' # the default
                  if $development;
 
+        mount '/getrate' => Plack::App::Proxy->new(remote => 'http://currencies.apps.grandtrunk.net/getrate')->to_app;
```

Comment:
This opens up the server as an anonymous proxy from anywhere on the web. I don't like that. Also, it does not keep a local cache (in the exchange rate table) for any values already requested.

This leads to inconsistencies (potentially) with the exchangerate table and the values returned by this service.